extension-jaxrs/src/main/java/io/smallrye/openapi/jaxrs/JaxRsAnnotationScanner.java

```diff
@@ -611,6 +614,22 @@ private Collection<ClassInfo> getJaxRsResourceClasses(AnnotationScannerContext c
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Use the Jandex index to find all resource classes specified in the configuration.
+     *
+     * @param context current scanning context
+     * @return Collection of ClassInfo's
+     */
+    private Collection<ClassInfo> getConfigurationResourceClasses(AnnotationScannerContext context) {
+        return context.getConfig().getScanResourceClasses()
+                .keySet()
+                .stream()
+                .map(key -> key.substring(OpenApiConstants.SCAN_RESOURCE_CLASS_PREFIX.length()))
```

Comment:
As mentioned above, `getScanResourceClasses` should return just the class names as the keys. That means the part here with `substring` won't be necessary.